.travis.yml

```diff
@@ -2,15 +2,30 @@ language: node_js
 node_js:
   - 7
 
+before_install:
+  - npm config set "//registry.npmjs.org/:_authToken=\${NPM_API_KEY}"
+
 before_script:
   - lerna bootstrap
 
 script:
   - npm test
 
 after_success:
-  - npm config set "//registry.npmjs.org/:_authToken=\${NPM_API_KEY}"
-  - script/cibuild --yes
+  - script/after_success
+
+deploy:
+  # publish release candidates on release branches
+  - provider: script
+    script: script/release-candidate
+    on:
+      branch: /^release/
+
+  # publish "final" releases on master
+  - provider: script
+    script: script/release
+    on:
+      branch: master
```

Comment:
Didn't realize you could have multiple deploys 🤘 Also glad to see it doesn't effect the `after_success` event.